packages/french-verbs/src/index.ts

```diff
@@ -127,26 +127,81 @@ export function alwaysAuxEtre(verb: string): boolean {
 }
 
 import listTransitive from 'french-verbs-transitive/dist/transitive.json';
-export function isTransitive(verb: string): boolean {
-  return listTransitive.indexOf(verb) > -1;
-}
 
-const validTenses: string[] = [
+export const validTenses = [

```

Comment:
is the list of tenses covered for French now 100% exhaustive? If so, please indicate it in a comment